### ISSUE TITLE
[Infra] Allow to programmatically scale actors on DevNet

### ISSUE THREAD

**Issue by okdas** [pokt-network/pocket#883]
## Objective

In order to thoroughly test the network, we need a way to programmatically provision and remove nodes on the network. This is a requirement for, for example, a state sync end-to-end test.

## Origin Document

Identified the requirement for this during #829.

## Goals

- A way to easily change infrastructure during end-to-end tests.

## Deliverable

- [x] An HTTP endpoint that accepts the name of the actor and the desired number of nodes.
- [x] As a consequence of that feature, we can scale network through the UI. Add that feature to the dashboard.
- [x] Update documentation to reflect the changes to scaling procedure, include information about HTTP endpoint and the UI. 

## General issue deliverables

- [ ] Update the appropriate CHANGELOG(s)
- [x] Update any relevant local/global README(s)
- [ ] Update relevant source code tree explanations
- [ ] Add or update any relevant or supporting [mermaid](https://mermaid-js.github.io/mermaid/) diagrams

## Testing Methodology

- [ ] **Task specific tests or benchmarks**: `make ...`
- [ ] **New tests or benchmarks**: `make ...`
- [ ] **All tests**: `make test_all`
- [ ] **LocalNet**: verify a `LocalNet` is still functioning correctly by following the instructions at [docs/development/README.md](https://github.com/pokt-network/pocket/tree/main/docs/development)
- [ ] **k8s LocalNet**: verify a `k8s LocalNet` is still functioning correctly by following the instructions [here](https://github.com/pokt-network/pocket/blob/main/build/localnet/README.md)

---

**Creator**: @okdas 
**Co-Owners**: N/A

**Comment by okdas** [comment]
I realized this has not been captured by the ticket on a dashboard, will add more details soon.

**Comment by okdas** [comment]
The feature has been implemented, I'm updating the documentation to reflect new changes.

**Comment by Olshansk** [comment]
> The feature has been implemented, I'm updating the documentation to reflect new changes.

Thanks for the update!

**Comment by Olshansk** [comment]
@okdas Can you please also update the main README in this repo to point to all the other docs / READMEs?

For example:
- [README.md](https://github.com/pokt-network/pocket/blob/main/README.md)
- [docs/development/README.md](https://github.com/pokt-network/pocket/blob/main/docs/development/README.md)
- [docs/learning/README.md](https://github.com/pokt-network/pocket/blob/main/docs/learning/README.md)

I'm 100% no one will complain about the redundancy of backlinks.

**Comment by Olshansk** [comment]
Also @okdas , did you get anyone else to review the documentation in the other places to make sure it works, is clear and functional?

**Comment by okdas** [comment]
Good call, I'm on it - I'll update the main repo README files as well. The dashboard has not been integrated with the LocalNet yet, though! That PR will be a good place to get feedback on documentation.
